README.md

```diff
@@ -54,7 +54,7 @@ Nothing changed [from before](https://github.com/CodeFX-org/demo-jigsaw-advent-c
 
 ### _chocolate_
 
-As [before with _factories_](https://github.com/CodeFX-org/demo-jigsaw-advent-calendar/tree/02-splitting-into-modules#factories) this module must `require public` the _surprise_ module.
+As [before with _factories_](https://github.com/CodeFX-org/demo-jigsaw-advent-calendar/tree/02-splitting-into-modules#factories) this module must `require transitive` the _surprise_ module.
```

Comment:
Since this is not the topic this branch is teaching, we can just drop the keywords and go with "...this module must require the _surprise_ module transitively".